test/unit/core/simd/view_to_simd_test.cpp

```diff
@@ -13,6 +13,7 @@
 
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/core/bit_manipulation.hpp>
+#include <seqan3/core/simd/debug_stream_simd.hpp>
 #include <seqan3/core/simd/simd_traits.hpp>
 #include <seqan3/core/simd/simd.hpp>
 #include <seqan3/core/simd/view_to_simd.hpp>
```

Comment:
Could you sort the seqan/std includes into the cpp std includes? (Sry, I can't mark them.)